src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User firstUser = new User("firstUser", "123456", 22);
+    private static final User secondUser = new User("secondUser", "myPassword", 16);
+    private static final User thirdUser = new User("thirdUser", "small", 19);
+    private static final User fourthUser = new User("fourthUser", "password", 18);
+    private static final User fifthUser = new User("fifthUser", null, 21);
+    private static final User sixthUser = new User("sixthUser", "password", null);
+    private static final User seventhUser = new User(null, "validPassword", 19);
+    private static final User eighthUser = new User("eighthUser", "validPassword", -1);
+    private static final User ninthUser = new User("small", "validPassword", 18);
+    private static final User sameFirstUser = new User("firstUser", "validPassword", 23);

```

Comment:
no need to make those users global, it would be ok to create a user in each test method 
this way it would be easy to understand and each test would have independent user, that wont be reused elsewhere